actors/evm/src/lib.rs

```diff
@@ -181,7 +181,7 @@ where
         }
         Outcome::Revert => Err(ActorError::unchecked_with_data(
             EVM_CONTRACT_REVERTED,
-            "contract reverted".to_string(),
+            format!("contract reverted at {0}", output.pc),
```

Comment:
We could consider adding additional information here. E.g., self destructed v. reverted? Or maybe the ID/address of the actor?